docs/io/configuration/components/models/converters/arepo_to_tardis.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>While this works I would recommend using:</p><p> </p><blockquote>&lt;div class="alert alert-info"&gt;</blockquote><blockquote>&nbsp;&nbsp;</blockquote><blockquote>Note</blockquote><blockquote><br></blockquote><blockquote>insert text of note here</blockquote><blockquote><br></blockquote><blockquote>&lt;/div&gt;</blockquote><p> </p><p>in a markdown cell (same for the other notes).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>